pandas/core/generic.py

```diff
@@ -6880,11 +6884,14 @@ def asof(self, where, subset=None):
         -------
         scalar, Series, or DataFrame
 
-           Scalar : when `self` is a Series and `where` is a scalar.
-           Series: when `self` is a Series and `where` is an array-like,
-             or when `self` is a DataFrame and `where` is a scalar.
-           DataFrame : when `self` is a DataFrame and `where` is an
-             array-like.
+            The return can be:
+
+            * Scalar : when `self` is a Series and `where` is a scalar
+            * Series: when `self` is a Series and `where` is an array-like,
+              or when `self` is a DataFrame and `where` is a scalar
+            * DataFrame : when `self` is a DataFrame and `where` is an
+              array-like
+            Return scalar, Sereis, or DataFrame.
```

Comment:
Does this render correctly? Thought a blank space was needed to separate the bullet points from the remaining text. 

Could be wrong just want to double check

Reply:
It is working fine. There is a space between the bullet and the text.